Source/SwiftLintBuiltInRules/Rules/Idiomatic/OneDelarationPerFileRule.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct OneDelarationPerFileRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.error)
```

Comment:
A warning should be sufficient:

```suggestion
    var configuration = SeverityConfiguration<Self>(.warning)
```

It can be configured as an error in the configuration later if preferred.